src/resources/favorites.ts

```diff
@@ -13,78 +13,9 @@ import { externalTokenQueryKey, fetchExternalToken } from './assets/externalAsse
 
 export const favoritesQueryKey = createQueryKey('favorites', {}, { persisterVersion: 2 });
```

Comment:
we can add to the persister version and create a new migration, or just wait as all token icons should show up after 24hrs as the stale time will trigger a refresh